app/controllers/peer_reviews_controller.rb

```diff
@@ -157,7 +157,21 @@ def assign_groups
         return
       end
     when 'unassign'
-      PeerReview.unassign(selected_reviewee_group_ids, reviewers_to_remove_from_reviewees_map)
+      reviews_deleted, deleted_count, undeleted_reviews = PeerReview.unassign(selected_reviewee_group_ids,
+                                                                              reviewers_to_remove_from_reviewees_map)
+      if !reviews_deleted && deleted_count == 0
```

Comment:
I don't think this case needs a separate message, you can merge it with the next one (length <= 5). See also my suggestion for rewording the message down below.